Plutarch/DataRepr.hs

```diff
@@ -66,6 +68,25 @@ type family PNames (as :: [PLabeled]) :: [Symbol] where
   PNames '[] = '[]
   PNames ((l ':= t) ': as) = l ': (PNames as)
 
+type family GetPDataRecordArgs (a :: PType) :: [[PLabeled]] where
```

Comment:
Since we don't export this anymore I don't think we need this alias anymore. You could just inline it.